src/single_units/unit.rs

```diff
@@ -1,56 +1,58 @@
-/// This structure represent all the stats that a unit can have.
-struct Stats {
-    /// The distance the model can Advance Move in inches per turn.
-    advance_rate: usize,
-    /// The distance the model can March Move in inches per turn.
-    march_rate: usize,
-    /// Represents the model’s overall morale, bravery and a unit’s ability to quickly reorganise during battle.
-    discipline: usize,
-    /// When the model loses this many Health Points, it is removed as a casualty
-    health_point: usize,
-    /// How well the model avoids being hit in melee.
-    defense: usize,
-    /// How easily the model withstands blows.
-    resilience: usize,
-    /// The Armour of the model.
-    armour: usize,
-    /// A special protection for the model, the final defense after the model’s Armour.
-    aegis: usize,
-    /// The number of Melee Attacks the model has.
-    attack: usize,
-    /// How good the model is at scoring hits in melee.
-    offensive: usize,
-    /// How easily the model can wound enemy models in melee.
-    strength: usize,
-    /// How well the model can penetrate the Armour of enemy models in melee.
-    armour_penetration: usize,
-    /// Models with higher Agility strike first in melee.
-    agility: usize,
-}
+pub mod math {
```

Comment:
No documentation for the module read : https://doc.rust-lang.org/rustdoc/what-is-rustdoc.html
I don't believe we want everything in there public, we want the Unit struct maybe Modifiers but not all the struct I think you should put `pub` in front of them instead of specifying the whole module is public